Lectures/Lecture8_OpenScienceReflections.md

```diff
@@ -1,41 +1,97 @@
-# Open Science for Physicists, Lecture [number] , [main topic]
+# Open Science for Physicists, Lecture 8 , Open Science Reflections
 
-*This is a template file, please make a copy in the ../Lectures director before customizing for a specific lecture*
-
-+ Last updated: DD MM 20YY
++ Last updated: 30 08 2024
 + Lecturer: e.g. @SanliFaez 
 
 #### Contributors: 
 + Erik van Sebille @erikvansebille
-+ Hosein Nemati
++ Tanja Hinderer
 
 ## Development goal
-__why are we covering this topic and how can this lecture help participants progress towards that learning goal __
-
+ To reflect on the key concepts in open science that we have discussed in this course. 
+ To practice storytelling and video production skills in creating a short (scientific) pitch.
+ 
 ## Motivation: 
-__sketch the context of the lecture, why is this topic timely or urgent to cover, and what is desired or unwanted in the status quo.__
+The last lecture of the course is an opportunity to reflect back on what has been discussed and learned, and what are the main take aways from this course. 
+The symposium format allows the participants to also create an overall impression of what standpoints their peers have developed.
 
 ## Vision of outcome
-__What skills and capabilities will the course participants obtain after finishing this lecture and the corresponding assignments?__
++ Participants choose a conscious position of how to practice open science in the next phases of our scientific career.
```

Comment:
'a conscious position' is perhaps a bit of a vague term here. Would 'consciously choose a position' or 'carefully choose a position' (or just 'choose a position') not be clearer?